src/js/events.js

```diff
@@ -16,11 +16,27 @@
         // Helpers for event handling
 
         attachDOMEvent: function (targets, event, listener, useCapture) {
+            var selectiveListener = function (event) {
```

Comment:
Was the goal of this feature to not emit _any_ DOM events on 'disabled' elements, or to just prevent any medium-editor behavior on 'disabled' elements?